phpunit.xml

```diff
@@ -14,7 +14,7 @@
         <!-- Example SIMPLETEST_BASE_URL value: http://localhost -->
         <env name="SIMPLETEST_BASE_URL" value="http://appserver/"/>
         <!-- Example SIMPLETEST_DB value: mysql://username:password@localhost/databasename#table_prefix -->
-        <env name="SIMPLETEST_DB" value="sqlite://appserver/sites/default/files/db.sqlite"/>
+        <env name="SIMPLETEST_DB" value="sqlite://appserver/:memory:"/>
```

Comment:
Ok had to look this one up, but assuming it's working correctly given the test suite passed!